### DISCUSSION TITLE
how to make capsule fall over?

### DISCUSSION THREAD

**Discussion by patrikpatrik** [DanielChappuis/reactphysics3d#269] [Q&A] [answered]
I'm tinkering with the capsule shape. But I'm noticing it doesn't fall over after I drop it in the world and stays perfectly straight up. Anyway to make it to fall realistically and roll around?

**Comment by DanielChappuis** [comment, marked as answer]
Hello,

You can try to use a very small rotation in your initial Transform when you create the body.
